docs/src/tutorials/topoplotseries.md

```diff
@@ -18,13 +18,14 @@ In case you do not already have data, you can get example data from the `TopoPlo
 You can do it like this:
 ```@example main
 data, positions = TopoPlots.example_data()
-df = UnfoldMakie.eeg_matrix_to_dataframe(data[:,:,1],string.(1:length(positions)));
+df = UnfoldMakie.eeg_matrix_to_dataframe(data[:,:,1], string.(1:length(positions)));
+insertcols!(df, 4, :channel =>  df[!, :label])
```

Comment:
maybe
`df.channel = df.label`
or even better
`rename(!df,:label => :channel)`?

or even better, change the topoplotfunction call so that it targets the column :label instead of :channel?

I'm a bit confused what the function plot_topoplotseries needs that column for anyway :S